inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -66,4 +69,38 @@ void ConvolutionLayerTest::SetUp() {
     ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(conv)};
     function = std::make_shared<ngraph::Function>(results, params, "convolution");
 }
+
+void ConvolutionLayerTest::GenerateInputs() {
+    const auto& inputsInfo = executableNetwork.GetInputsInfo();
+    const auto& functionParams = function->get_parameters();
+    for (int i = 0; i < functionParams.size(); ++i) {
+        const auto& param = functionParams[i];
+        const auto infoIt = inputsInfo.find(param->get_friendly_name());
+        GTEST_ASSERT_NE(infoIt, inputsInfo.cend());
+
+        const auto& info = infoIt->second;
+        auto blob = GenerateInput(*info);
+        inputs.push_back(blob);
+    }
+}
+
+InferenceEngine::Blob::Ptr ConvolutionLayerTest::GenerateInput(const InferenceEngine::InputInfo& info) const {
+    return FuncTestUtils::createAndFillBlob(
```

Comment:
Should be redefined in 'LayerTestCommon'

Reply:
Yes, LayerTestsCommon::GenerateInput is fixed.